### PR TITLE
Milestone 2: Update Animals Table

### PR DESCRIPTION
**Pull request by leopiresgoss**
## Milestone 2: Update Animals Table

In this milestone I:

- Added a column species of type string to your animal's table
- Inserted more animals data
- Added transactions to update the table
- Wrote queries to answer the questions:
  - How many animals are there?
  - How many animals have never tried to escape? 
  - What is the average weight of animals?
  - Who escapes the most, neutered or not neutered animals?
  - What is the minimum and maximum weight of each type of animal?
  - What is the average number of escape attempts per animal type of those born between 1990 and 2000?

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -15,3 +15,86 @@ SELECT * from animals WHERE neutered = TRUE;
...
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/* TRANSACTIONS */
+
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+ROLLBACK;
```
Comment (DeliceLydia) [inline queries.sql]: - [x] You are supposed to check if the change was made you can simply do:
 `SELECT species from animals` to verify that change was made
ROLLBACK;

 `SELECT species from animals`; verify that change was undone

File: queries.sql [hunk 2]
```diff
@@ -15,3 +15,86 @@ SELECT * from animals WHERE neutered = TRUE;
...
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+COMMIT;
+
```
Comment (DeliceLydia) [inline queries.sql:50]: - [x] Before committing you can simply check if the change was made
`SELECT species from animals;`
COMMIT
`SELECT species from animals;` To check if change persists after commit

File: queries.sql [hunk 3]
```diff
@@ -15,3 +15,86 @@ SELECT * from animals WHERE neutered = TRUE;
...
+SAVEPOINT SP1;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+
+ROLLBACK TO SAVEPOINT SP1;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
```
Comment (DeliceLydia) [inline queries.sql:76]: - [x] Before this query Add a query that `UPDATE` all animals' weight to be their weight multiplied by -1.
`UPDATE animals SET weight_kg = weight_kg * -1;`
then Rollback to the savepoint
`SAVEPOINT SP1;`

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @leopiresgoss, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Highlights for your project:
- Awesome README
- Followed git workflow


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: queries.sql (hunk 3 above)
Comment (leopiresgoss) [reply queries.sql:76]: I have done that [lines 59 - 62](https://github.com/leopiresgoss/vet-clinic-db/blob/9c7bdd8c011f52f96c892d11fbb978eaf1424bac/queries.sql#L59-L62)

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @leopiresgoss,

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**